Source/WebCore/rendering/RenderSelection.cpp

```diff
@@ -48,7 +48,7 @@ namespace WebCore {
 namespace {
 
 struct SelectionContext {
-    
```

Comment:
You should update your IDE to avoid this noise in PRs. We prefer not to touch spaces on lines we don't update.